src/app/hm-admin/services/models/Property.ts

```diff
@@ -11,9 +11,9 @@ export class Property {
       this.label = datas[definition.title] || '';
       this.type = datas[definition.property] ? datas[definition.property]['@type'] : '';
       this.range = datas[definition.range] || '';
```

Comment:
Careful, I haven’t verified if you are up to date with master branch, but this `definition.range` is not the one we want (see #41)